gtsfm/two_view_estimator.py

```diff
@@ -94,6 +92,13 @@ def create_computation_graph(
             Two view report w/ verifier metrics wrapped as Delayed.
             Two view report w/ post-processor metrics wrapped as Delayed.
         """
+        # Unpack GT data.
+        if gt_gtsfm_data is not None:
+            gt_wTi1_graph = gt_gtsfm_data.get_camera(0).pose()
```

Comment:
instead of hard-coding camera 0 and 1, we should pull out `i1` and `i2` here

Reply:
If so, I would have to pass in `i1` and `i2`.

Reply:
I'm good with passing in i1 and i2. I think whenever we use a camera index in GTSFM, we need to be able to assume it really means the index we expect

Reply:
I don't see the utility in passing in two indices for a single object as opposed to just using a known mapping (i.e., i1 -> 0 and i2 -> 1), especially since none of the other objects use the indices.

Reply:
I partially agree with John. I think it might be best to accept gt_i1Ti2 and the gt_mesh as two arguments. 
Passing `i1` and `i2` I think makes the API too messy. But yes, as you said (also in another comment), assuming the input has only 2 cameras at 0 and 1 could lead to bugs.